master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/index.md

```diff
@@ -32,70 +32,78 @@ You must have a cluster which meets the following requirements:
 
 This document describes three installation options for Calico using Kubernetes API as the datastore:
 
--  Calico policy with Calico networking
--  Calico policy-only
-   -  with user-supplied networking
-   -  with flannel networking
+1. Calico policy with Calico networking (beta)
+2. Calico policy-only with user-supplied networking
+3. Calico policy-only with flannel networking
 
 Ensure you have a cluster which meets the above requirements.  There may be additional requirements based on the installation option you choose.
 
-> Note:  There is currently no upgrade path to switch between different installation options.  Therefore, 
-> if you are upgrading from Calico v2.1, use the [Calico policy-only with user-supplied networking](#calico-policy-only-with-user-supplied-networking) installation instructions 
+> Note:  There is currently no upgrade path to switch between different installation options.  Therefore,
+> if you are upgrading from Calico v2.1, use the [Calico policy-only with user-supplied networking](#calico-policy-only-with-user-supplied-networking) installation instructions
 > to upgrade Calico policy-only which leaves the networking solution unchanged.
 
-### Calico policy with Calico networking (Beta)
+### 1. Calico policy with Calico networking (Beta)
 
 With Kubernetes as the Calico datastore, Calico has Beta support for Calico networking.  This provides BGP-based
 networking with a full node-to-node mesh.  It is not currently possible to configure the Calico BGP network to peer with
 other routers - future releases of Calico are expected to bring feature parity with the etcd-backed Calico.
 
 To install Calico with Calico networking, run one of the following commands based on your Kubernetes version:
 
-For Kubernetes 1.6 clusters:
+For **Kubernetes 1.6** clusters:
 
 ```
 kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.6/calico.yaml
 ```
 
 >[Click here to view the above yaml directly.](calico-networking/1.6/calico.yaml)
 
-For Kubernetes 1.5 clusters:
+For **Kubernetes 1.5** clusters:
 
 ```
 kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.5/calico.yaml
 ```
 
 >[Click here to view the above yaml directly.](calico-networking/1.5/calico.yaml)
 
-### Calico policy-only with user-supplied networking
```

Comment:
This diff is misleading. You'll see at the bottom of the added section this header is still here unaltered (besides adding the number 2)